compiler.go

```diff
@@ -1287,6 +1320,28 @@ func (c *Compiler) printTrace(a ...interface{}) {
 	_, _ = fmt.Fprintln(c.trace, a...)
 }
 
+func (c *Compiler) getPathModule(moduleName string) (pathFile string, err error) {
```

Comment:
I know it's not directly related to this particular PR, but now that we allow changing the module file extension, I wonder if we should allow specifying the extension explicitly in the code. For example.

```golang
mod := import("my_module.some_ext")
```

should work even if `.some_ext` is not in `importFileExt` slice. What do you think?

Reply:
I’d be fine with that but personally I think it’s a bit clunky, and creates two ways of doing imports where I’d rather have one.

Reply:
ok. that's fair. let's not do that for now. also if we allow extensions in the code it kind of breaks backward compatibility now that I think about it.

Reply:
I fixed the failing test @ 2d816c3. Plz, re-run the workflow when you have time.